client/src/components/PublicPages/AuditPlanner/ElectionResultsCard.tsx

```diff
@@ -0,0 +1,295 @@
+                              name: e.target.value,
+                            })
+                          }
+                          placeholder="Candidate name"
+                          value={candidate.name}
+                        />
+                      </CandidateContainer>
+                    ) : (
+                      candidate.name
+                    )}
+                  </td>
+                  <td>
+                    {editable ? (
+                      <VotesContainer>
+                        <IntegerInput
+                          ariaLabel={`Candidate ${i} Votes`}
+                          name={`candidate[${i}].votes`}
+                          onChange={newValue =>
+                            setCandidate(i, {
+                              ...candidate,
+                              votes: newValue,
+                            })
+                          }
+                          value={candidate.votes}
+                        />
+                      </VotesContainer>
+                    ) : (
+                      candidate.votes
+                    )}
+                  </td>
+                </tr>
+              ))}
+              {editable && (
+                <tr>
+                  <td>
+                    <Button icon="plus" onClick={addCandidate}>
+                      Add Candidate
+                    </Button>
+                  </td>
+                  <td />
+                </tr>
+              )}
+            </tbody>
+          </CandidatesTable>
+
+          <AdditionalInputsRow>
+            <AdditionalInputContainer>
+              <label>
+                <span>Number of Winners</span>
+                {editable ? (
+                  <IntegerInput
+                    name="numberOfWinners"
+                    onChange={newValue =>
+                      setElectionResults({
+                        ...electionResults,
+                        numWinners: newValue,
+                      })
+                    }
+                    value={numWinners}
+                  />
+                ) : (
+                  numWinners
+                )}
+              </label>
+            </AdditionalInputContainer>
+            <AdditionalInputContainer>
+              <label>
+                <span>Total Ballots Cast</span>
+                {editable ? (
+                  <IntegerInput
+                    name="totalBallotsCast"
+                    onChange={newValue =>
+                      setElectionResults({
+                        ...electionResults,
+                        totalBallotsCast: newValue,
+                      })
+                    }
+                    value={totalBallotsCast}
+                  />
+                ) : (
+                  totalBallotsCast
+                )}
+              </label>
+            </AdditionalInputContainer>
+          </AdditionalInputsRow>
+        </InnerContainer>
+
+        <CardActionsRow>
```

Comment:
Now that I'm seeing this button bar implemented, the buttons feel way too big lol. I'd like to try out some other options

Reply:
Switched to our more standard right-aligned buttons (see the updated screen capture in the PR description)

Reply:
Nice, those look better